war/src/main/scss/base/_display.scss

```diff
@@ -1,3 +1,7 @@
 .jenkins-\!-display-contents {
   display: contents;
 }
+
+.jenkins-\!-display-none {
```

Comment:
there's also a `jenkins-hidden` class in the public API is there a reason to add a new one?